utils/swift_build_support/swift_build_support/products/ninja.py

```diff
@@ -18,7 +18,7 @@
 import platform
 import sys
 
-from . import product
+from . import product, product_builder
```

Comment:
I usually prefer to keep separate module imports on separate lines. Grouping usually indicates to me that we're importing multiple symbols specifically from a module.